.travis.yml

```diff
@@ -9,5 +9,3 @@ script:
   - npm run ci
 after_script:
   - npminstall codecov && codecov
```

Comment:
try 
```
rm -rf node_modules
npm install
```

Reply:
what does it mean?

Reply:
Try run those commands at the root of this project, it will auto generate the .yml files.